src/Core/Models/Domain/PasswordGeneratorPolicyOptions.cs

```diff
@@ -2,12 +2,31 @@
 {
     public class PasswordGeneratorPolicyOptions
     {
+        public string DefaultType { get; set; }
         public int MinLength { get; set; }
         public bool UseUppercase { get; set; }
         public bool UseLowercase { get; set; }
         public bool UseNumbers { get; set; }
         public int NumberCount { get; set; }
         public bool UseSpecial { get; set; }
         public int SpecialCount { get; set; }
+        public int MinNumberOfWords { get; set; }
+        public bool Capitalize { get; set; }
+        public bool IncludeNumber { get; set; }
+        
+        public bool InEffect()
+        {
+            return DefaultType != ""
```

Comment:
Formatting done elsewhere in the codebase is to include `||`'s and `&&`'s at the end of the line. For example:

```
return DefaultType != "" ||
    Condition1 ||
    Condition2 ||
    ...
```

Reply:
Updated!